include/odp/api/spec/queue.h

```diff
@@ -44,6 +45,32 @@ extern "C" {
  */
 odp_queue_t odp_queue_create(const char *name, const odp_queue_param_t *param);
 
+/**
+ * Create multiple queues
+ *
+ * Otherwise like odp_queue_create(), but creates multiple queues with a single
+ * call. The output queue handles are written in the same order as input
+ * parameters. A single odp_queue_create_multi() call is equivalent to calling
+ * odp_queue_create() 'num' times in row.
+ *
+ * If 'share_param' value is false, 'param' array must contain 'num' elements.
+ * If the value is true, only a single element is required. If 'name' array is
```

Comment:
This could highlight the purpose of the flag (share the same params): "If the value is true, only a single element is required and it is used as queue parameters for all created queues."

Reply:
Added in V2.